CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
   [thiagohmcruz](https://github.com/thiagohmcruz)
   [#401](https://github.com/CocoaPods/CocoaPods/issues/401)
 
-* Bump Xcode version constants for Xcode 12.3  
+* Bump Xcode version constants for Xcode 12.4
```

Comment:
Lets add a separate entry instead of editing the existing one. Also please keep the two empty spaces at the end for markdown formatting.

Reply:
Okay 👍🏽  Fwiw this entry had already been changed from Xcode 12 -> 12.3